crates/red_knot_python_semantic/src/semantic_index/re_exports.rs

```diff
@@ -26,7 +30,20 @@ use rustc_hash::FxHashSet;
 
 use crate::{module_name::ModuleName, resolve_module, Db};
 
-#[salsa::tracked(return_ref)]
+fn symbol_cycle_recover(
+    _db: &dyn Db,
+    _value: &FxHashSet<Name>,
+    _count: u32,
+    _file: File,
+) -> salsa::CycleRecoveryAction<FxHashSet<Name>> {
+    salsa::CycleRecoveryAction::Iterate
+}
+
+fn symbol_cycle_initial(_db: &dyn Db, _file: File) -> FxHashSet<Name> {
+    FxHashSet::default()
+}
```

Comment:
These names are generally related to the query name, so e.g. `exported_cycle_recover` and `exported_cycle_initial`